test/components/admin/editions/show/sidebar_actions_component_test.rb

```diff
@@ -25,6 +25,17 @@ class Admin::Editions::Show::SidebarActionsComponentTest < ViewComponent::TestCa
     assert_selector "a", text: "View on website (opens in new tab)"
   end
 
+  test "actions for published edition for non-english document" do
+    current_user = build_stubbed(:user)
+    edition = create(:non_english_published_edition)
+    render_inline(Admin::Editions::Show::SidebarActionsComponent.new(edition:, current_user:))
+
+    assert_selector "li", count: 3
+    assert_selector "button", text: "Create new edition"
+    assert_selector "a", text: "View data about page"
+    assert_selector "a[href='#{edition.public_url(locale: edition.primary_locale)}']", text: "View on website (opens in new tab)"
```

Comment:
Would it be possible to change this to check for the actual link, e.g. `/government/somewhere.pt` (otherwise we're only checking we call the right thing, rather than testing the actual link is correct)?

Also could you add the check for the `href` value to the test above, to ensure we're not appending a locale for the English only edition.